shared/src/business/useCases/docketEntry/addPaperFilingInteractor.ts

```diff
@@ -190,12 +190,19 @@ export const addPaperFiling = async (
   let paperServicePdfUrl;
 
   if (isReadyForService) {
+    const currentDocketEntry = caseEntities[0].getDocketEntryById({
+      docketEntryId,
+    });
+    const electronicParties =
+      currentDocketEntry.eventCode === 'ATP' ? [] : undefined;
+
```

Comment:
I'm trying to understand why this is necessary.

Reply:
For event code ATP the method where we are sending in this variable requires a new process when sending email notification, and so to keep other interactors that use this method the same as before and not implement many if conditions we pass in this new electronicParties parameter and if it is undefined it works as usual but if we pass it in it uses it to override who gets an email notification.